models/models_methods_test.go

```diff
@@ -112,6 +112,16 @@ func TestCheckDuplicatedTriggerMapItems(t *testing.T) {
 }
 
 func TestTriggerMapItemModelString(t *testing.T) {
+	t.Log("triggering pipeline")
```

Comment:
I see some `trigger-check` integration tests in the codebase, I think we should extend those too.